benchmarking/benchmarks.jl

```diff
@@ -1,260 +1,346 @@
-        :ode => @ODEmodel(
-            EGFR'(t) = EGFR_turnover * pro_EGFR(t) + EGF_EGFR(t) * reaction_1_k2 - EGFR(t) * EGFR_turnover - EGF_EGFR(t) * reaction_1_k1,
-            pEGFR'(t) = EGF_EGFR(t) * reaction_9_k1 - pEGFR(t) * reaction_4_k1 + pEGFR_Akt(t) * reaction_2_k2 + pEGFR_Akt(t) * reaction_3_k1 - Akt(t) * pEGFR(t) * reaction_2_k1,
-            pEGFR_Akt'(t) = Akt(t) * pEGFR(t) * reaction_2_k1 - pEGFR_Akt(t) * reaction_3_k1 - pEGFR_Akt(t) * reaction_2_k2,
-            Akt'(t) = pAkt(t) * reaction_7_k1 + pEGFR_Akt(t) * reaction_2_k2 - Akt(t) * pEGFR(t) * reaction_2_k1,
-            pAkt'(t) = pAkt_S6(t) * reaction_5_k2 - pAkt(t) * reaction_7_k1 + pAkt_S6(t) * reaction_6_k1 + pEGFR_Akt(t) * reaction_3_k1 - S6(t) * pAkt(t) * reaction_5_k1,
-            S6'(t) = pAkt_S6(t) * reaction_5_k2 + pS6(t) * reaction_8_k1 - S6(t) * pAkt(t) * reaction_5_k1,
-            pAkt_S6'(t) = S6(t) * pAkt(t) * reaction_5_k1 - pAkt_S6(t) * reaction_6_k1 - pAkt_S6(t) * reaction_5_k2,
-            pS6'(t) = pAkt_S6(t) * reaction_6_k1 - pS6(t) * reaction_8_k1,
-            EGF_EGFR'(t) = EGF_EGFR(t) * reaction_1_k1 - EGF_EGFR(t) * reaction_9_k1 - EGF_EGFR(t) * reaction_1_k2,
-            y1(t) = a1 * (pEGFR(t) + pEGFR_Akt(t)),
-            y2(t) = a2 * (pAkt(t) + pAkt_S6(t)),
-            y3(t) = a3 * pS6(t)
-        )
-    ),
-    Dict(
-        :name => "CD8 T cell differentiation",
-        :ode => @ODEmodel(
-            N'(t) = - N(t) * mu_N - N(t) * P(t) * delta_NE,
-            E'(t) = N(t) * P(t) * delta_NE - E(t)^2 * mu_EE - E(t) * delta_EL + E(t) * P(t) * rho_E,
-            S'(t) = S(t) * delta_EL - S(t) * delta_LM - S(t)^2 * mu_LL - E(t) * S(t) * mu_LE,
-            M'(t) = S(t) * delta_LM - mu_M * M(t),
-            P'(t) = P(t)^2 * rho_P - P(t) * mu_P - E(t) * P(t) * mu_PE - S(t) * P(t) * mu_PL,
-            y1(t) = N(t),
-            y2(t) = E(t) + S(t),
-            y3(t) = M(t)
-        )
-    ),
-    Dict(
-        :name => "Chemical reaction network",
-        :ode => @ODEmodel(    
-            x1'(t) = -k1 * x1(t) * x2(t) + k2 * x4(t) + k4 * x6(t),
-            x2'(t) = -k1 * x1(t) * x2(t) + k2 * x4(t) + k3 * x4(t),
-            x3'(t) = k3 * x4(t) + k5 * x6(t) - k6 * x3(t) * x5(t),
-            x4'(t) = k1 * x1(t) * x2(t) - k2 * x4(t) - k3 * x4(t),
-            x5'(t) = k4 * x6(t) + k5 * x6(t) - k6 * x3(t) * x5(t),
-            x6'(t) = -k4 * x6(t) - k5 * x6(t) + k6 * x3(t) * x5(t),
-            y1(t) = x3(t),
-            y2(t) = x2(t)
-        )
-    )
+    Dict(:name => "Modified LV for testing",
+         :ode => @ODEmodel(x1'(t)=(a + b) * x1(t) - c * x1(t) * x2(t),
+                           x2'(t)=-a * b * x2(t) + d * x1(t) * x2(t),
+                           y1(t)=x1(t))),
+    Dict(:name => "SIWR original",
+         :ode => @ODEmodel(function S'(t)
+                               mu - bi * S(t) * I(t) - bw * S(t) * W(t) - mu * S(t) +
+                               a * R(t)
+                           end,
```

Comment:
Why do we do this?

Reply:
oh, that was an accidental; formatter thinks this is a function definition. I will fix that